sphinx_rtd_theme/layout.html

```diff
@@ -25,7 +25,7 @@
 
   {#- CSS #}
   {%- if sphinx_version_info < (4, 0) -%}
-    <link rel="stylesheet" href="{{ pathto('_static/' + style, 1) }}" type="text/css" />
+    <link rel="stylesheet" href="{{ pathto('_static/' + styles[-1], 1) }}" type="text/css" />
```

Comment:
@polyzen - thanks for opening this. However, the line that you are changing is only be relevant for `if sphinx_version_info < (4, 0)`.

I think https://github.com/readthedocs/sphinx_rtd_theme/issues/1465 was closed because the bug was finally identified to have another origin?

Reply:
Brainfart.